### PR TITLE
Fix unexpected keyword argument 'split' in issue #438

### LINKED ISSUE #438: TypeError: _get_tsv_paths() got an unexpected keyword argument 'split'
Hi,
clinicadl version 1.3.1, python 3.8, miniconda environment, I am having the error below with the command of training. Currently able to continue training with the same command below with the earlier version clinicadl 1.2.0, python 3.8, pytorch 1.13.1.
training command:
clinicadl train classification /truba/home/meatay/ADNIMRI_CAPS/ /truba/home/meatay/ADNIMRI_CAPS/tensor_extraction/image.json \
                /truba/home/meatay/T1Wlabels/ADCN/ADCNsplit/ /truba/home/meatay/DLrun/ADCNDLimage/ \
                --config_file /truba/home/meatay/train_config.toml
error log:
Traceback (most recent call last):
  File "/truba/home/meatay/miniconda3/envs/ClinicaDL/bin/clinicadl", line 8, in <module>
    sys.exit(cli())
  File "/truba/home/meatay/miniconda3/envs/ClinicaDL/lib/python3.8/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/truba/home/meatay/miniconda3/envs/ClinicaDL/lib/python3.8/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/truba/home/meatay/miniconda3/envs/ClinicaDL/lib/python3.8/site-packages/click/core.py", line 1657, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/truba/home/meatay/miniconda3/envs/ClinicaDL/lib/python3.8/site-packages/click/core.py", line 1657, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/truba/home/meatay/miniconda3/envs/ClinicaDL/lib/python3.8/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/truba/home/meatay/miniconda3/envs/ClinicaDL/lib/python3.8/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/truba/home/meatay/miniconda3/envs/ClinicaDL/lib/python3.8/site-packages/clinicadl/train/tasks/classification_cli.py", line 78, in cli
    task_launcher("classification", task_specific_options, **kwargs)
  File "/truba/home/meatay/miniconda3/envs/ClinicaDL/lib/python3.8/site-packages/clinicadl/train/tasks/task_utils.py", line 110, in task_launcher
    train(Path(kwargs["output_maps_directory"]), train_dict, train_dict.pop("split"))
  File "/truba/home/meatay/miniconda3/envs/ClinicaDL/lib/python3.8/site-packages/clinicadl/train/train.py", line 14, in train
    maps_manager = MapsManager(maps_dir, train_dict, verbose=None)
  File "/truba/home/meatay/miniconda3/envs/ClinicaDL/lib/python3.8/site-packages/clinicadl/utils/maps_manager/maps_manager.py", line 86, in __init__
    self._check_args(parameters)
  File "/truba/home/meatay/miniconda3/envs/ClinicaDL/lib/python3.8/site-packages/clinicadl/utils/maps_manager/maps_manager.py", line 1222, in _check_args
    train_df = split_manager[0]["train"]
  File "/truba/home/meatay/miniconda3/envs/ClinicaDL/lib/python3.8/site-packages/clinicadl/utils/split_manager/split_manager.py", line 115, in __getitem__
    train_df, valid_df = self.concatenate_diagnoses(item)
  File "/truba/home/meatay/miniconda3/envs/ClinicaDL/lib/python3.8/site-packages/clinicadl/utils/split_manager/split_manager.py", line 129, in concatenate_diagnoses
    train_path, valid_path = self._get_tsv_paths(
TypeError: _get_tsv_paths() got an unexpected keyword argument 'split'

### PR DESCRIPTION
**Pull request by camillebrianceau**

### CODE REVIEW

File: clinicadl/utils/split_manager/single_split.py [hunk 1]
```diff
@@ -30,7 +30,7 @@ def allowed_splits_list(self):
     def split_iterator(self):
         return range(1)
 
-    def _get_tsv_paths(self, cohort_path: Path):
+    def _get_tsv_paths(self, cohort_path: Path, split):
```
Comment (ravih18) [inline clinicadl/utils/split_manager/single_split.py]: The variable `split` is not used the function.

Comment (camillebrianceau) [reply clinicadl/utils/split_manager/single_split.py]: Yes but it is used in the `kfold_split` for the same function so it is needed to avoid errors !

Comment (ravih18) [reply clinicadl/utils/split_manager/single_split.py]: I was thinking that there might be a better way to it, without passing "ghost" argument. But if it is easier that way, it is fine.

**Review by ravih18 — APPROVED** [review]

**Review by ravih18 — COMMENTED** [review]
This looks better

**Pull request merged**